python/tvm/relay/backend/contrib/uma/uma_cli.py

```diff
@@ -0,0 +1,92 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+    UMA Command Line Interface (CLI)
```

Comment:
We probably need to discuss about this - (I think we missed this in the RFC)
We need this documented in the guide-level explanation and the rationale to create a bespoke CLI should be discussed, IMO. 

cc : @leandron 

Also, I think it would be beneficial break this component to another PR for smoother upstreaming of things.

Reply:
@manupa-arm, I agree in general.
IMHO uma_cli is more a helper for the tutorial in its current form, therefore I think it could be of use for early users of UMA. 
A deeper embedding into TVM, e.g. via tvmc is definitely a further discussion that we need to have, e.g. on the RFC level

Reply:
In that case, Would it be better to move it from here to tutorials or apps/ until we have the discussion how to integrate to back to main compiler ?

Reply:
Just talked to @cgerum. We would tend to move it to apps